CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## In Development
 
+## v0.16.0
+* st2web now uses HTTP by default (#72)
```

Comment:
Ideally to add a few more words that it's now recommended to rely on "LoadBalancer" or "Ingress" to add HTTPs layer on top of it.

Reply:
That's a good idea. Thanks.